src/bridge/MatrixHandler.ts

```diff
@@ -856,13 +857,14 @@ export class MatrixHandler {
                         ircRoom.server, event.sender, displayName
                     );
                     await this.sendIrcAction(req, ircRoom, bridgedClient, ircAction, event);
+                    return null;
```

Comment:
Why is this now returning `Promise<null>` instead of `Promise<undefined>` into `messageSendPromiseSet`?

Reply:
leftover from me trying something out, removed.